src/Instrumentation/CodeIgniter/src/CodeIgniterInstrumentation.php

```diff
@@ -116,6 +116,14 @@ public static function register(): void
                         $span->setStatus(StatusCode::STATUS_ERROR);
                     }
 
+                    // Propagate server-timing header to response, if ServerTimingPropagator is present
+                    if (class_exists('OpenTelemetry\Contrib\Propagation\ServerTiming\ServerTimingPropagator')) {
```

Comment:
I think this and subsequent FQN strings would be safer if it was a `use` statement with `::class`. I don't think that should cause any issues (except maybe with static analysis).

Reply:
Maybe that would make more sense once this has been extracted to some common module? Current approach is more copy-paste friendly, and automatic tests do not cover this if the class cannot be found due to a missing `use`. If one were to copy this to a new instrumentation and forget the `use`, the only way they find out that it doesn't work is with manual testing with the propagator included as a dependency and seeing that the header is missing.